tests/attacks/test_projected_gradient_descent.py

```diff
@@ -294,8 +291,240 @@ def test_scikitlearn(self):
             # Check that x_test has not been modified by attack and classifier
             self.assertAlmostEqual(float(np.max(np.abs(x_test_original - self.x_test_iris))), 0.0, delta=0.00001)
 
-    def test_classifier_type_check_fail(self):
-        backend_test_classifier_type_check_fail(ProjectedGradientDescent, [BaseEstimator, LossGradientsMixin])
+    @unittest.skipIf(tf.__version__[0] != '2', '')
+    def test_framework_tensorflow_v2_mnist(self):
+        classifier, _ = get_image_classifier_tf()
+
+        scores = get_labels_np_array(classifier.predict(self.x_train_mnist))
+        acc = np.sum(np.argmax(scores, axis=1) == np.argmax(self.y_train_mnist, axis=1)) / self.y_train_mnist.shape[0]
+        logger.info("[TFV2, MNIST] Accuracy on training set: %.2f%%", acc * 100)
+
+        scores = get_labels_np_array(classifier.predict(self.x_test_mnist))
+        acc = np.sum(np.argmax(scores, axis=1) == np.argmax(self.y_test_mnist, axis=1)) / self.y_test_mnist.shape[0]
+        logger.info("[TFV2, MNIST] Accuracy on test set: %.2f%%", acc * 100)
```

Comment:
I think we can remove these tests, because they are not testing PGD.

Reply:
Done.